apps/oc_chef_wm/priv/dispatch-chef-wm.conf

```diff
@@ -1,63 +0,0 @@
-%%-*- mode: erlang -*-
-%% ex: ft=erlang ts=4 sw=4
-%% Copyright 2012 Opscode, Inc. All Rights Reserved.
-%%
-%% This file is provided to you under the Apache License,
-%% Version 2.0 (the "License"); you may not use this file
-%% except in compliance with the License.  You may obtain
-%% a copy of the License at
-%%
-%%   http://www.apache.org/licenses/LICENSE-2.0
-%%
-%% Unless required by applicable law or agreed to in writing,
-%% software distributed under the License is distributed on an
-%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-%% KIND, either express or implied.  See the License for the
-%% specific language governing permissions and limitations
-%% under the License.
-%%
-
-
-{["cookbooks"], chef_wm_cookbooks, []}.
```

Comment:
They were moved into dispatch.conf a while back - this file was just never cleaned up during the merge of chef_wm into oc_chef_wm